src/Counter/Counter.js

```diff
@@ -0,0 +1,32 @@
+import React, { createElement, useState } from "react";
+
+class Counter extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      counter: 0
+    };
+  }
+
+  handleIncrement = () => {
+    this.setState({ counter: this.state.counter + 1 });
+  };
+
+  handleDecrement = () => {
+    this.setState({ counter: this.state.counter - 1 });
+  };
+  
+  render() {
+    return createElement(
+      "div",
+      null,
+      createElement("button", { className: "btn btn-primary", onClick: this.handleIncrement }, "Increment"),
+      createElement("h1", null, this.state.counter),
+      createElement("button", { className: "btn btn-primary", onClick: this.handleDecrement }, "Decrement"),
+    );
+  }
+}
+
+export default Counter
```

Comment:
let's not use default export, it's not a good approach https://rajeshnaroth.medium.com/avoid-es6-default-exports-a24142978a7a

instead of it you can use:

```
export class Counter extends React.Component {
...
```

Reply:
fixed